front/app/containers/Admin/initiatives/messages.ts

```diff
@@ -117,4 +113,8 @@ export default defineMessages({
     id: 'app.containers.Admin.initiatives.enableProposals',
     defaultMessage: 'Enable proposals',
   },
+  proposals: {
+    id: 'app.containers.Admin.initiatives.proposals',
+    defaultMessage: 'Proposals',
```

Comment:
The same as in the previous PR, do we need to remove the old "Proposals" copy?

Reply:
I'm used to these being picked up by the dead code check. Let me take a look

Reply:
And maybe the "Proposals" copy from the sidebar should be removed too.

Reply:
Ah sorry, we still use it.